src/content/docs/en/extend/add-content-collections.mdx

```diff
@@ -0,0 +1,326 @@
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+import Since from '~/components/Since.astro'
+import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro'
+import RecipeLinks from "~/components/RecipeLinks.astro"
+
+
+**Content collections** are a powerful way to manage groups of similar content, such as blog posts. Collections help to organize your documents, validate your YAML frontmatter, and provide automatic TypeScript type-safety for all of your content (even if you don't write any TypeScript yourself).
+
+## Prerequisites
+
+You will need **an existing Astro project with Markdown or MDX files in the `src/pages/` folder**.
+
+This tutorial uses the [Build a Blog tutorial's finished project](https://github.com/withastro/blog-tutorial-demo) to demonstrate converting a blog to content collections. You can fork and use that codebase locally, or complete the tutorial in the browser by [editing the blog tutorial's code in StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).
+
+You can instead follow similar steps in your own Astro project, but you will need to adjust the instructions for your own codebase. We recommend using our sample project to complete this short tutorial first. Then, you can use what you have learned to work on your own codebase.
+
+## Build a Blog Tutorial Code
+
+In the Build a Blog introductory tutorial, you learned about Astro's [built-in file-based routing](/en/core-concepts/routing/#static-routes): any `.astro` or `.md` file anywhere within the `src/pages/` folder automatically became a page on your site.
+
+To create your first blog post at `https://example.com/posts/post-1/`, you created a `/posts/` folder and added the file `post-1.md`. You then added a new Markdown file to this folder every time you wanted to add a new blog post to your site.
+
+
+## Pages vs Collections
+
+Even when using content collections, you will still use the `src/pages/` folder for individual pages, such as your About Me page. But, moving your blog posts to the special `src/content/` folder will allow you to use more powerful and performant APIs to generate your blog post index and display your individual blog posts.
+
+At the same time, you'll receive better guidance and autocompletion in your code editor because you will have a **[schema](/en/guides/content-collections/#defining-a-collection-schema)** to define a common structure for each post that Astro will help you enforce. In your schema, you can specify when frontmatter properties are required, such as a description or an author, and which data type each property must be, such as a string or an array. The leads to catching many mistakes sooner, with descriptive error messages telling you exactly what the problem is.
```

Comment:
Small typo
```suggestion
At the same time, you'll receive better guidance and autocompletion in your code editor because you will have a **[schema](/en/guides/content-collections/#defining-a-collection-schema)** to define a common structure for each post that Astro will help you enforce. In your schema, you can specify when frontmatter properties are required, such as a description or an author, and which data type each property must be, such as a string or an array. This leads to catching many mistakes sooner, with descriptive error messages telling you exactly what the problem is.
```